pkg/sql/sem/tree/testdata/pretty/20.align-deindent.golden

```diff
@@ -253,40 +211,53 @@ CREATE TABLE t (
 CREATE TABLE t (
 	a INT8, b INT8, c INT8,
 	PRIMARY KEY (a, b)
-) INTERLEAVE IN PARENT p2 (i)
-  PARTITION BY LIST (a) (
+) PARTITION BY LIST (a) (
 							PARTITION p1 VALUES IN (1),
 							PARTITION p2 VALUES IN (2)
                         )
 
-80:
---------------------------------------------------------------------------------
-CREATE TABLE t (
-	a INT8, b INT8, c INT8,
-	PRIMARY KEY (a, b)
-) INTERLEAVE IN PARENT p2 (i)
-  PARTITION BY LIST (a) (PARTITION p1 VALUES IN (1), PARTITION p2 VALUES IN (2))
+79:
+-------------------------------------------------------------------------------
+CREATE TABLE t (a INT8, b INT8, c INT8, PRIMARY KEY (a, b)) PARTITION BY LIST (
+																a
+                                                            ) (
+																PARTITION p1
```

Comment:
this looks worse than the deleted diff. has there been a regression? I feel like foundations should confirm